content-usable/index.html

```diff
@@ -692,17 +692,10 @@ <h2>User Stories</h2>
           <h3>Objective 1: Help Users Understand What Things are and How to Use Them</h3>
 
 
-			<h5>Clear Purpose</h5>
-      <p>Persona:</p>
-      <ul>
-        <li><a href="#scenario-3-losing-focus-when-completing-tasks" aria-label="Carolyn: Losing focus when completing tasks">Carolyn: Focus</a></li>
-        <li><a href="" aria-label="">Frank: Making a doctors appointment</a></li>
-        <li><a href="" aria-label="">Maria: Pressing the correct button</a></li>
-        <li><a href="" aria-label="">Maria: Dynamic website elements make key website information difficult to locate</a></li>
-        <li><a href="" aria-label="">Tom: Understanding where information is in a hierarchical structure</a></li>
-      </ul> 
+			<h5>User Stories For Clear Purpose</h5>
```

Comment:
@lseeman @rachaelbradley  It's not clear what this should be. In every case except 1, these subheadings only have a single User Story.  There are always multiple "User Needs". Perhaps:

<h5>User Story: Clear Purpose
  <blockquote>As a user with a memory impairment, attention.......
This includes the following User Needs:

* I know what the website offers....
* ....